src/data/balance/pokemon-evolutions.ts

```diff
@@ -88,64 +90,71 @@ interface PokemonEvolutions {
 }
 
+/** TODO: Should maybe have multiple tiers for each of these */
+const GENERIC_ITEM_EVO_LEVEL = 36;
+const HAPPINESS_EVO_LEVEL = 25;
+const KNOW_MOVE_EVO_LEVEL = 36;
+
 export const pokemonEvolutions: PokemonEvolutions = {
-  [Species.BULBASAUR]: [new SpeciesEvolution(Species.IVYSAUR, 16, null, null)],
-  [Species.IVYSAUR]: [new SpeciesEvolution(Species.VENUSAUR, 32, null, null)],
-  [Species.CHARMANDER]: [new SpeciesEvolution(Species.CHARMELEON, 16, null, null)],
-  [Species.CHARMELEON]: [new SpeciesEvolution(Species.CHARIZARD, 36, null, null)],
-  [Species.SQUIRTLE]: [new SpeciesEvolution(Species.WARTORTLE, 16, null, null)],
-  [Species.WARTORTLE]: [new SpeciesEvolution(Species.BLASTOISE, 36, null, null)],
-  [Species.CATERPIE]: [new SpeciesEvolution(Species.METAPOD, 7, null, null)],
-  [Species.METAPOD]: [new SpeciesEvolution(Species.BUTTERFREE, 10, null, null)],
-  [Species.WEEDLE]: [new SpeciesEvolution(Species.KAKUNA, 7, null, null)],
-  [Species.KAKUNA]: [new SpeciesEvolution(Species.BEEDRILL, 10, null, null)],
-  [Species.PIDGEY]: [new SpeciesEvolution(Species.PIDGEOTTO, 18, null, null)],
-  [Species.PIDGEOTTO]: [new SpeciesEvolution(Species.PIDGEOT, 36, null, null)],
-  [Species.RATTATA]: [new SpeciesEvolution(Species.RATICATE, 20, null, null)],
-  [Species.SPEAROW]: [new SpeciesEvolution(Species.FEAROW, 20, null, null)],
-  [Species.EKANS]: [new SpeciesEvolution(Species.ARBOK, 22, null, null)],
-  [Species.SANDSHREW]: [new SpeciesEvolution(Species.SANDSLASH, 22, null, null)],
-  [Species.NIDORAN_F]: [new SpeciesEvolution(Species.NIDORINA, 16, null, null)],
-  [Species.NIDORAN_M]: [new SpeciesEvolution(Species.NIDORINO, 16, null, null)],
-  [Species.ZUBAT]: [new SpeciesEvolution(Species.GOLBAT, 22, null, null)],
-  [Species.ODDISH]: [new SpeciesEvolution(Species.GLOOM, 21, null, null)],
-  [Species.PARAS]: [new SpeciesEvolution(Species.PARASECT, 24, null, null)],
-  [Species.VENONAT]: [new SpeciesEvolution(Species.VENOMOTH, 31, null, null)],
-  [Species.DIGLETT]: [new SpeciesEvolution(Species.DUGTRIO, 26, null, null)],
-  [Species.MEOWTH]: [new SpeciesFormEvolution(Species.PERSIAN, "", "", 28, null, null)],
-  [Species.PSYDUCK]: [new SpeciesEvolution(Species.GOLDUCK, 33, null, null)],
-  [Species.MANKEY]: [new SpeciesEvolution(Species.PRIMEAPE, 28, null, null)],
-  [Species.POLIWAG]: [new SpeciesEvolution(Species.POLIWHIRL, 25, null, null)],
-  [Species.ABRA]: [new SpeciesEvolution(Species.KADABRA, 16, null, null)],
-  [Species.MACHOP]: [new SpeciesEvolution(Species.MACHOKE, 28, null, null)],
-  [Species.BELLSPROUT]: [new SpeciesEvolution(Species.WEEPINBELL, 21, null, null)],
-  [Species.TENTACOOL]: [new SpeciesEvolution(Species.TENTACRUEL, 30, null, null)],
-  [Species.GEODUDE]: [new SpeciesEvolution(Species.GRAVELER, 25, null, null)],
-  [Species.PONYTA]: [new SpeciesEvolution(Species.RAPIDASH, 40, null, null)],
+  [Species.BULBASAUR]: [new SpeciesEvolution(Species.IVYSAUR, 16, null, null, 0)],
```

Comment:
```suggestion
  [Species.BULBASAUR]: [new SpeciesEvolution(Species.IVYSAUR, 16, null, null)],
```
Same with all the others that pass 0 in.